esmvaltool/recipes/recipe_flato13ipcc.yml

```diff
@@ -600,6 +606,382 @@ diagnostics:
+      - {<<: *cmip3, dataset: inmcm3_0, institute: INM}
+      - {<<: *cmip3, dataset: ipsl_cm4, institute: IPSL}
+      - {<<: *cmip3, dataset: miroc3_2_hires, institute: NIES}
+      - {<<: *cmip3, dataset: miroc3_2_medres, institute: NIES}
+      # - {<<: *cmip3, dataset: miub_echo_g, institute: MIUB-KMA}
+      - {<<: *cmip3, dataset: mpi_echam5, institute: MPIM}
+      # - {<<: *cmip3, dataset: mri_cgcm2_3_2a, institute: MRI}
+      - {<<: *cmip3, dataset: ncar_ccsm3_0, institute: NCAR}
+      - {<<: *cmip3, dataset: ncar_pcm1, institute: NCAR}
+      - {<<: *cmip3, dataset: ukmo_hadcm3, institute: UKMO}
+      - {<<: *cmip3, dataset: ukmo_hadgem1, institute: UKMO}
+
+      - {dataset: ACCESS1-0}
+      - {dataset: ACCESS1-3}
+      - {dataset: bcc-csm1-1}
+      - {dataset: bcc-csm1-1-m}
+      - {dataset: BNU-ESM}
+      - {dataset: CanCM4}
+      - {dataset: CanESM2}
+      - {dataset: CCSM4}
+      - {dataset: CESM1-BGC}
+      - {dataset: CESM1-CAM5}
+      - {dataset: CESM1-FASTCHEM}
+      - {dataset: CESM1-WACCM}
+      - {dataset: CMCC-CESM}
+      - {dataset: CMCC-CM}
+      - {dataset: CMCC-CMS}
+      - {dataset: CNRM-CM5}
+      - {dataset: CNRM-CM5-2}
+      - {dataset: FIO-ESM}
+      - {dataset: GFDL-CM2p1}
+      - {dataset: GFDL-CM3}
+      - {dataset: GFDL-ESM2G}
+      - {dataset: GFDL-ESM2M}
+      - {dataset: GISS-E2-H}
+      - {dataset: GISS-E2-H-CC}
+      - {dataset: GISS-E2-R}
+      - {dataset: GISS-E2-R-CC}
+      - {dataset: HadCM3}
+      - {dataset: HadGEM2-AO}
+      - {dataset: HadGEM2-CC}
+      - {dataset: HadGEM2-ES}
+      - {dataset: inmcm4}
+      - {dataset: IPSL-CM5A-LR}
+      - {dataset: IPSL-CM5A-MR}
+      - {dataset: IPSL-CM5B-LR}
+      - {dataset: MIROC-ESM}
+      - {dataset: MIROC-ESM-CHEM}
+      - {dataset: MIROC4h}
+      - {dataset: MIROC5}
+      - {dataset: MPI-ESM-LR}
+      - {dataset: MPI-ESM-MR}
+      - {dataset: MPI-ESM-P}
+      - {dataset: MRI-CGCM3}
+      - {dataset: MRI-ESM1}
+      - {dataset: NorESM1-M}
+      - {dataset: NorESM1-ME}
+
+      - {dataset: CERES-EBAF, project: obs4mips, level: L3B, version: Ed2-7,
+         tier: 1, start_year: 2003, end_year: 2011}
+    scripts:
+      fig09-6_pattern_cor:
+        <<: *fig09_6_pattern_cor
+
+  fig09-6_pr:
+    description: Calculate pattern correlation for precipitation
+    variables:
+      pr:
+        preprocessor: regrid_4_5
+        reference_dataset: GPCP-SG
+        alternative_dataset: GHCN
+        project: CMIP5
+        exp: historical
+        ensemble: r1i1p1
+        mip: Amon
+        field: T2Ms
+        start_year: 1980
+        end_year: 1999
+    additional_datasets:
+      - {<<: *cmip3, dataset: bccr_bcm2_0, institute: BCCR}
+      - {<<: *cmip3, dataset: cccma_cgcm3_1, institute: CCCMA}
+      - {<<: *cmip3, dataset: cccma_cgcm3_1_t63, institute: CCCMA}
+      - {<<: *cmip3, dataset: csiro_mk3_0, institute: CSIRO}
+      # - {<<: *cmip3, dataset: gfdl_cm2_0, institute: GFDL}
```

Comment:
Why all the commented out datasets?

Reply:
Data available but filenames have a dot before .nc and ESMValTool doesn't find them

Reply:
It would be good to report that in the pull request that adds support for CMIP3 to ESMValCore, so it can be fixed there.